tools/underlay.repos

```diff
@@ -23,7 +23,19 @@ repositories:
     type: git
     url: https://github.com/stevemacenski/bond_core.git
     version: fix_deprecation_warning
+  ros/diagnostics:
+    type: git
+    url: https://github.com/ros/diagnostics.git
```

Comment:
Why diagnostics?  I imagine geographic info is a dep of RL, is diagnostics too?

Reply:
yes thats also RL's dependency
https://github.com/cra-ros-pkg/robot_localization/blob/79162b2ac53a112c51d23859c499e8438cf9938e/package.xml#L27